docs/usage/framework-agnostic-php-project.md

```diff
@@ -607,6 +607,46 @@ try {
 }
 ```
 
+### Handling exceptions
+
+You can use Ray to handle exceptions using the `try` and `catch` functions.  These methods work in a similar manner to the way that `try...catch` blocks do in PHP.
+
+- The `try` function accepts a single `$ray` parameter which may be used to access the current `Ray` instance.
+- The `catch` function accepts either no parameters _(see below)_ or two parameters: `$ray` and `$exception`, which are the current `Ray` instance and the caught `Exception`, respectively.
+
+```php
+function myfunc($value) {
+    if ($value > 5) {
+        throw new \Exception('hello exception ' . $value);
+    }
+    
+    return 'hello world ' . $value;
+}
+
+foreach(range(1, 10) as $num) {
+    ray()->try(function($ray) use ($num) {
```

Comment:
I'm thinking we should use the most simple example here. I feel that using the passed `$ray` instance is rather exotic. Let's just use the `ray()` function in the callbacks to send stuff to Ray.